Makefile

```diff
@@ -87,7 +87,7 @@ lint: venv ## Check Python code style compliance
 	@which flake8 > /dev/null || pip install flake8
 	@flake8 sdk/python --show-source --statistics \
 		--select=E9,E2,E3,E5,F63,F7,F82,F4,F841,W291,W292 \
-		--per-file-ignores sdk/python/tests/compiler/testdata/*:F841 \
+		--per-file-ignores sdk/python/tests/compiler/testdata/*:F841,F821 \
```

Comment:
Flake8 has some issue to detect the `NamedTuple` type name. Since F821 is only about Python style, we can ignore it in our testdata only. 
https://stackoverflow.com/questions/37840142/how-to-avoid-flake8s-f821-undefined-name-when-has-been-installed-by-get